modules/azure_devops_agent_container_app_jobs/main.tf

```diff
@@ -0,0 +1,24 @@
+
+locals {
+  additional_ips = [for ip in var.kv_ip_rules : "${ip}/32"]
+  ip_rules       = concat(local.additional_ips, ["${chomp(data.http.host_ip.response_body)}/32"])
+  default_tags = {
+    module = "altinn/altinn-modules/altinn_azure_devops_agent_container_app_jobs"
```

Comment:
I suppose this was supposed to be **org/repo/module** but the name became too long? 😄

Reply:
Based on the module source ref in terraform (but removed the obvious azurerm part and duplicate modules ref)
https://registry.terraform.io/modules/Altinn/altinn-modules/azurerm/latest/submodules/github_runner_container_app_jobs